selfdrive/controls/lib/events.py

```diff
@@ -635,6 +635,10 @@ def joystick_alert(CP: car.CarParams, CS: car.CarState, sm: messaging.SubMaster,
     ET.NO_ENTRY: wrong_car_mode_alert,
   },
 
+  EventName.resumeBlocked: {
+    ET.NO_ENTRY: NoEntryAlert("Press Set to Engage"),
+  },
```

Comment:
This seemed fine on the Honda, it's very similar to our current no entry alerts (main is off, gear not D, etc)